packages/core/src/cache.ts

```diff
@@ -24,6 +24,10 @@ function md5(buffer: BinaryLike): string {
 }
 
 export function createCacheKey(options: ResolvedConfigOptions) {
+  if (options.cacheKey) {
```

Comment:
This was on purpose, so cacheKey would be included with all other options, not just as a single key. Like, what if you don't change the cacheKey but do change other options? Then the cache key does not represent the config anymore.

Reply:
Oh good point! See new commit!